README.md

```diff
@@ -1,31 +1,15 @@
-# PlaceHolderKV documentation
+# Valkey documentation
 
 ## Clients
 
-All clients are listed under language specific sub-folders of [clients](./clients)
-
-The path follows the pattern: ``clients/{language}/github.com/{owner}/{repository}.json``.
-The ``{language}`` component of the path is the path-safe representation
-of the full language name which is mapped in [languages.json](./languages.json).
-
-Each client's JSON object represents the details displayed on the [clients documentation page](https://placeholderkv.io/docs/clients).
-
-For example [clients/python/github.com/placeholderkv](./clients/python/github.com/placeholderkv/placeholderkv-py.json):
-
-```
-{
-    "name": "placeholderkv-py",
-    "description": "Mature and supported. Currently the way to go for Python.",
-    "recommended": true
-}
-```
+This section is under construction.
```

Comment:
Can't we keep the existing clients?

Their information is contributed to the doc repo (creative commons share alike) so we are allowed to keep them imo. we can remove the "official" tag on all of them though.

Reply:
Agreed. will revert and add clarification.